src/solve_quadratic.cpp

```diff
@@ -1,26 +1,14 @@
 /*!
  *
  * \author Andrey Krotov
- * \file main.cpp
+ * \file solve_quadratic.h
  *
  */
 
-#include <stdio.h>
-#include <assert.h>
-#include <math.h>
-#include <float.h>
-
-/*! \enum
- * Possible number of roots of quadratic equations.
- */
-
-enum numberOfRoots {
-    INF_ROOTS = -1, /**< infinite roots  */
-    NO_ROOTS,       /**< no       roots  */
-    ONE_ROOT,       /**< one      root   */
-    TWO_ROOTS       /**< two      roots  */
-};
+#include "solve_quadratic.h"
+#include "test_quadratic.cpp"
```

Comment:
Move main() to a separate file and don't include test_quadratic in this file (and don't include .cpp files!!!).